assets/app/scripts/services/buildConfigs.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+angular.module("openshiftConsole")
+  .factory("BuildConfigsService", function(){
```

Comment:
@spadgett forgot to include the screenshot:
![screenshot-7](https://cloud.githubusercontent.com/assets/1668218/12689535/c725ba2c-c6dd-11e5-980d-73a84e989d69.png)